src/Handlers/AbstractPaymentMethodHandler.php

```diff
@@ -62,13 +62,15 @@
 use Shopware\Core\System\SalesChannel\SalesChannelContext;
 use Symfony\Component\HttpFoundation\RedirectResponse;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Session\Session;
 use Symfony\Component\Routing\RouterInterface;
 use Symfony\Component\Security\Csrf\CsrfTokenManagerInterface;
 
 abstract class AbstractPaymentMethodHandler
 {
 
     const PROMOTION = 'promotion';
+    const SAFE_ERROR_CODES = ['124'];
```

Comment:
Could we add a comment here pointing to the docs where the error codes are documented? So that it can be easily maintained